manifests/charts/gateway/templates/service.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     topology.istio.io/network: "{{.}}"
     {{- end }}
   annotations:
-    {{- .Values.annotations | toYaml | nindent 4 }}
+    {{- .Values.service.annotations | toYaml | nindent 4 }}
```

Comment:
Do we want both `.Values.service.annotations` and `.Values.annotations` here @howardjohn? Same goes for other resources in this chart. Description of the annotations field:

```yaml
# Annotations to apply to all resources
annotations: {}
```

Reply:
I think we need both, I see another PR for this: https://github.com/istio/istio/pull/36189